index.html

```diff
@@ -1,19 +1,131 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
+    <title>My Resume</title>
+    <link rel="stylesheet" href="css/index.css" />
+    <!--[if lt IE 9]>
```

Comment:
Don't forget the `<meta name="viewport" content="width=device-width, initial-scale=1.0">`

Reply:
Sure i won't due to my bad network connectivity I couldn't make the site responsive